fixtures/features/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@fixtures/features",
   "version": "0.0.13",
-  "main": "src/index.ts",
```

Comment:
Is this because Parcel treats the HTML file as the entry point?

Reply:
Yeah, and it complains about this in that case.